### PR TITLE
[FIX] Leak mne cov matrix

### PR DESCRIPTION
**Pull request by juangpc**

### CODE REVIEW

**Comment by juangpc** [comment]
This should be ready for review

File: libraries/mne/c/mne_cov_matrix.cpp [hunk 1]
```diff
@@ -162,10 +177,10 @@ int mne_decompose_eigen (double *mat,
     double *dmat = MALLOC_30(np,double);
     float  *vecp = vectors[0];
 
-    const char   *uplo  = "U";
-    const char   *compz = "V";
+//    const char   *uplo  = "U";
+//    const char   *compz = "V";
```
Comment (gabrielbmotta) [inline libraries/mne/c/mne_cov_matrix.cpp:181]: If these are unused, should we just delete them entirely instead of commenting them out?

Comment (juangpc) [reply libraries/mne/c/mne_cov_matrix.cpp:181]: agree

**Review by gabrielbmotta — APPROVED** [review]
👍

**Pull request merged**